fictadvisor-web/src/components/pages/admin/questions-admin-page/components/questions-search-page/components/questions-table/components/table-actions/TableActions.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const tableColumn: SxProps<Theme> = {
+  flexDirection: 'row',
+  gap: '8px',
+  alignItems: 'center',
+};
```

Comment:
forgot `display: flex`

Reply:
we don`t need a display: flex here, as there is a Stack from mui, which have display: flex by default